### PR TITLE
PASS-1296: Create Vault envvars on haproxy nodes

### PR DESCRIPTION
**Pull request by amitc-jahia**
JIRA issue: https://jira.jahia.org/browse/PAAS-1296

Short description:
Create Vault envvars on haproxy nodes at env installation

### CODE REVIEW

File: install.yml [hunk 1]
```diff
@@ -26,6 +26,9 @@ globals:
   shortdomain: ${settings.shortdomain}
   displayName: ${settings.displayName}
   jahia_docker_image: ${settings.jahiaDockerImage}
+  vault_roleid: ${settings.vaultRoleId}
+  vault_secretid: ${settings.vaultSecretId}
+  vault_clusterurl: ${settings.vaultClusterUrl}
```
Comment (kevinvandel) [inline install.yml:31]: The settings need to be declared under `settings:` (at the end of the file)
(note: they need to be declared as "required")

**Review by kevinvandel — CHANGES_REQUESTED** [review]

File: install.yml (hunk 1 above)
Comment (laurentfufu) [reply install.yml:31]: about the `required`, maybe we shouldn't while the front don't sent them ?

**Review by laurentfufu — CHANGES_REQUESTED** [review]

File: install.yml (hunk 1 above)
Comment (kevinvandel) [reply install.yml:31]: Damn, good point !
So either we do it in two steps (2 PR), or we merge this PR and the one on the frontend side at the same time (may not be easy to synchronize)

Comment (amitc-jahia) [reply install.yml:31]: Requested changes have been made. The settings have been declared under `settings`. Please Review.

**Review by kevinvandel — APPROVED** [review]

**Review by laurentfufu — APPROVED** [review]

**Pull request merged**